doc/code_style/mom.cpp

```diff
@@ -230,17 +223,18 @@ bool YourMom::tick() {
 }
 
 // Unused parameters are commented out:
-void YourMom::buy(const Thing & /*item*/) {
+void YourMom::buy(const Thing& /*item*/) {
```

Comment:
like above, the `&` should be attached to the variable name. One or no space between `&` and the comment would be ok.